tasks/computer-vision/image-classification/iMat-Fashion/README.md

```diff
@@ -0,0 +1,21 @@
+Dataset Details
+
+
+Apparel instance segmentations include 27 main apparel objects (jackets, dresses, skirts, etc) and 19 apparel parts (sleeves, collars, etc). A total of 92 fine-grained attributes was annotated by expert for main apparel objects.
+In this dataset, you are provided a large number of images and corresponding fashion/apparel segmentations. Images are named with a unique ImageId. You must segment and classify the images in the test set.
+
+All of the test set images have fine-grained segmentations. The training set contains images both with and without fine-grained class segmentations.
+
+This dataset contains images of people wearing a variety of clothing types in a variety of poses. It may have content that some consider sensitive. 
+
+
+Dataset: https://www.datasetlist.com/

```

Comment:
@aiSynergy37 Change this line to: `https://www.kaggle.com/c/imaterialist-fashion-2019-FGVC6/data`